src/Microsoft.Health.Fhir.Core.UnitTests/Features/Operations/Import/CancelImportRequestHandlerTests.cs

```diff
@@ -66,14 +66,26 @@ public async Task GivenAFhirMediator_WhenCancelingExistingBulkImportJobThatHasAl
             Assert.Equal(HttpStatusCode.Conflict, operationFailedException.ResponseStatusCode);
         }
 
+        [Theory]
+        [InlineData(JobStatus.Cancelled)]
+        public async Task GivenAFhirMediator_WhenCancelingExistingBulkImportJobThatHasAlreadyBeenCanceled_ThenAcceptedStatusCodeShouldBeReturned(JobStatus taskStatus)
```

Comment:
This is new behavior, because if a tastk has been already canceled, then we will return accepted because we will continue trying to cancel.

Reply:
A change in behavior may require giving customers 1 month notice before deploying to prod. We should check w./ PM team